repo2docker/buildpacks/base.py

```diff
@@ -42,10 +42,17 @@
 ENV USER ${NB_USER}
 ENV HOME /home/${NB_USER}
 
-RUN adduser --disabled-password \
-    --gecos "Default user" \
-    --uid ${NB_UID} \
-    ${NB_USER}
+RUN groupadd \
+        --gid ${NB_UID} \
+        ${NB_USER} &&
```

Comment:
```suggestion
        ${NB_USER} && \
```

in the CI log one of the errors came up as `docker.errors.APIError: 400 Client Error: Bad Request ("Dockerfile parse error line 33: unknown instruction: USERADD")` which made me spot this.